selectBoard.php

```diff
@@ -77,6 +78,38 @@
         $stmt = $db->prepare($query); 
        	$result = $stmt->execute($query_params);   	
 
+
+$query = " 
+          INSERT INTO taskList ( 
+              listTitle, 
+              boardID
+          ) VALUES ( 
+              :listTitle, 
+              :boardID
+          ) 
+          "; 
+        $query_params = array( 
+          ':listTitle' => 'todo',
+          ':boardID' => $id
+        ); 
+        $stmt = $db->prepare($query); 
+       	$result = $stmt->execute($query_params);    	
+
+$query = " 
+         INSERT INTO taskList ( 
+             listTitle, 
+             boardID
+         ) VALUES ( 
+             :listTitle, 
+             :boardID
+         ) 
+         "; 
+       $query_params = array( 
+         ':listTitle' => 'done',
+         ':boardID' => $id
+       ); 
+       $stmt = $db->prepare($query); 
+        $result = $stmt->execute($query_params); 
```

Comment:
could be a loop that iterates twice and runs the query 2 times. right now there is some repetition that makes the page longer.

Reply:
okay